File: social-service/src/test/java/com/eirsteir/coffeewithme/social/web/api/user/UserControllerTest.java
package com.eirsteir.coffeewithme.social.web.api.user;

import com.eirsteir.coffeewithme.config.EventuateTestConfig;
import com.eirsteir.coffeewithme.social.config.ModelMapperConfig;
import com.eirsteir.coffeewithme.social.security.SecurityConfig;
import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
import org.springframework.context.annotation.Import;
import org.springframework.test.context.ActiveProfiles;

@ActiveProfiles("test")
@Import({SecurityConfig.class, ModelMapperConfig.class, EventuateTestConfig.class})
@WebMvcTest(UserController.class)
class UserControllerTest {}

File: social-service/src/test/kotlin/com/eirsteir/coffeewithme/social/web/api/user/UserControllerTest.kt
package com.eirsteir.coffeewithme.social.web.api.user

import com.eirsteir.coffeewithme.config.EventuateTestConfig
import org.springframework.context.annotation.Import

@ActiveProfiles("test")
@Import(SecurityConfig::class, ModelMapperConfig::class, EventuateTestConfig::class)
@WebMvcTest(
    UserController::class
)
internal class UserControllerTest 